l10n_us_check_writing_address/report/report_check_base_bottom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_check_base_bottom">
+        <t t-call="report.html_container">
+            <t t-foreach="docs" t-as="o">
+                <div class="page">
+                    <br/>
+                    <div>
+                        <strong>
+                            <span t-field="o.partner_id.name"/>
+                            <span style="padding-right:60mm;float:right;"
+                                  t-esc="o.payment_date"/>
+                        </strong>
+                        <t t-foreach="range(2)" t-as="i">
+                            <table width="100%"
+                                   style="padding-right:22mm;">
+                                <thead>
+                                    <tr style="text-align:left;">
+                                        <th style="padding-top:3mm;">Due
+                                            Date
+                                        </th>
+                                        <th>Description</th>
+                                        <th>Original Amount</th>
+                                        <th>Balance Due</th>
+                                        <th>Payment</th>
+                                    </tr>
+                                </thead>
+                                <tbody>
+                                    <t t-foreach="paid_lines[o.id]"
+                                       t-as="line">
+                                        <tr style="text-align:left;border-top: 0px;">
+                                            <td style="padding-top:3mm;">
+                                                <span t-esc="line['date_due']"/>
+                                            </td>
+                                            <td>
+                                                <span t-esc="line['reference'] or line['number']"/>
```

Comment:
@murtuzasaleh Can you display both fields with the `reference` field first and then the `number` on all format?